### PR TITLE
Insert the TeamView session link into the reply box

### PR DESCRIPTION
**Pull request by zpawn**
Story https://app.shortcut.com/deskpro/story/86067/teamviewer-insert-link

### CODE REVIEW

**Review by iTweeno — APPROVED** [review]

File: src/pages/HomePage.tsx [hunk 1]
```diff
@@ -181,14 +95,28 @@ const HomePage: FC = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [client]);
 
+    const onInsertLink = useCallback((sessionLink) => {
+        if (client) {
+            client?.deskpro().send({
+                type: "append_to_active_ticket_reply_box",
```
Comment (AshleyDawson) [inline src/pages/HomePage.tsx:101]: Please use the `client.deskpro().appendLinkToActiveTicketReplyBox()` method to add the link, as per the docs: https://support.deskpro.com/en-US/guides/developers/deskpro-ui

**Review by AshleyDawson — CHANGES_REQUESTED** [review]

File: src/pages/HomePage.tsx (hunk 1 above)
Comment (AshleyDawson) [reply src/pages/HomePage.tsx:101]: @zpawn  please use the session URL for both the link text and URL

Comment (AshleyDawson) [reply src/pages/HomePage.tsx:101]: Also, don't think you need the if block if you're coalescing the call, i.e. `client?.deskpro()...`

Comment (zpawn) [reply src/pages/HomePage.tsx]: Everything fixed

**Review by AshleyDawson — APPROVED** [review]

**Pull request merged**